docs/src/how-to/install/oauth.md

```diff
@@ -0,0 +1,72 @@
+# OAuth
+
+If not configured, OAuth is disabled per default. OAuth can be enabled in the wire-server Helm as follows:
+
+```yaml
+brig:
+  # ...
+  config:
+    # ...
+    optSettings:
+      # ...
+      setOAuthEnabled: true
+```
+
+To use the OAuth functionality, you will need to set up a public and private JSON web key pair (JWK) in the wire-server Helm chart. This key pair will be used to sign and verify OAuth access tokens.
+
+To configure the JWK, go to the wire-server Helm chart and provide the JWK information, as shown in the example below:
+
+```yaml
+# values.yaml or secrets.yaml
+brig:
+  secrets:
+    oauthJwkKeyPair: |
+      {
+        "p":"8U9gI_...",
+        "kty":"RSA",
+        "q":"43dqC...",
+        "d":"ixZk7x...",
+        "e":"AQAB",
+        "use":"sig",
+        "kid":"QVapB_J...",
+        "qi":"sYHbPsy...",
+        "dp":"LFmnVNPW...",
+        "alg":"RS256",
+        "dq":"UXTY7...",
+        "n":"1mnyGVT..."
+      }
+```
+
+Note that the JWK is a sensitive configuration value, so it is recommended to use Helm's support for managing secrets instead of including it in a plaintext values.yaml file.
```

Comment:
maybe reference how that support works, especially how it works in other places in this project?

Reply:
I currently don't know how this works. What about a `FUTURWORK: fill in the details on how this works once we have implemented it for prod`?

Reply:
This is specific to an individual deployment, and not even limited to the oauth secrets, but other secrets as well. It might be fine to explicitly point it out like that here, but how secrets are handled really is a case-by-case thing, and whoever deploys wire should form their own informed decision.